### ISSUE TITLE
How to a custom plugin capture a process

### ISSUE THREAD

**Issue by tuannp** [xjsframework/xjs#152]
Hello guys,
I wanna to make a plugin, that capture a process and display content like a screen capture.
How can i do it ?

**Comment by Matrim-Cauthon** [comment]
Hi, can you clarify what you are trying to accomplish here? Are you trying to create a source plugin to display something in your presentation? Is it something that cannot be done with simple Screen Capture, or Window Capture?

**Comment by tuannp** [comment]
Thanks you for quick response.
My problem is when i add a Screen Capture with a process, that includes 2 window (Main window and Sub window), i selected Window Source is the Sub. It work fine.
However, if hide the Sub window the Screen Capture switch to select the Main window automatically.
Then i intend to make a custom plugin like what i did into OBS Studio, that capture only a window source via title. 
I just found Stick to Title option is way to solve my problem :D
Sorry about limited knowledge question 
===
One other question, 
I got once time, the Screen Capture displays content from other window. The target window shows at bottom of other. But i can't reproduce it anymore, so strange!
